DashKit/DashPumpManager.swift

```diff
@@ -854,6 +854,9 @@ open class DashPumpManager: PumpManager {
         let enactUnits = roundToSupportedBolusVolume(units: units)
         
         guard let bolus = try? Bolus(immediateVolume: Int(round(enactUnits * Pod.podSDKInsulinMultiplier))) else {
+            self.mutateState({ (state) in
```

Comment:
I think this works. Might be more organized to put `Bolus` construction into the preflight check; the preflight could be changed to return a `Result<Bolus, PumpManagerError>`.  But only if you agree.